Code/Tools/AssetProcessor/native/resourcecompiler/JobsModel.cpp

```diff
@@ -335,7 +335,13 @@ namespace AssetProcessor
                 return message;
             }
             case JobStatus::InProgress:
-                return tr("InProgress");
+                QString message = tr("InProgress");
+                QString extra;
+
+                Append(message, critical ? "Critical" : "", " ");
+                Append(message, escalation != 0 ? QStringLiteral("(%1)").arg(escalation) : "", " ");
```

Comment:
It feels like we don't want the `Critical` or `Escalation` as part of the status - we want it in the message itself. That way users can filter on Status for all In-Progress jobs without assigning them a specific status code based on multiple priorities, and then they can further filter (or only filter) on messages containing a `(Critical)` or `(High-priority)` marker.

Reply:
I'm not sure I follow.  There's no description column and the message field here is just the name of the variable which holds the data that's displayed in the Status column.  The filter system only works on the status column, so if we move this info to another column you wouldn't be able to filter on it.  I'm not sure how useful it is to filter beyond "InProgress" though, its difficult to work with the job list while AP is processing things.  If you filter to InProgress and try to find some file or set of files, the list will be constantly moving and rearranging, with items appearing and disappearing rapidly.

I can move this status info into its own column easy enough, just realize the info is not stored anywhere so once a job exits "InProgress" status, the column will go blank, which could be confusing to users.

Reply:
Understood - what I'm asking for may be a future improvement then, rather than something that can be achieved right now. This was more of an "ideal state", meaning we may want to add a form of "priority" column or report on current asset processing status via live updates in a message area. @yuyihsu @amzn-leenguy  is this already a known area of improvement for UX as part of our asset processing message initiative?